apps/sms/test/marionette/lib/views/dialer.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+var View = require('./view');
+var Dialer =  require('../../../../../../apps/communications/dialer/test/marionette/lib/dialer.js');
+
+function DialerWrapper(client) {
+  this.client = client;
+}
+module.exports = DialerWrapper;
+
+DialerWrapper.prototype = {
+  __proto__: View.prototype,
+
+  switchToFrame: function() {
```

Comment:
We had this discussion with @julienw earlier, so we chose ```app.switchTo``` instead, but not sure if it sounds better for you